### PR TITLE
PD-6969 added stubbed fetch metadata module to warden

### PR DESCRIPTION
**Pull request by rvanburen-r7**
Added a stubbed fetch metadata function to warden.

this will be completed when metadata is in S3

### CODE REVIEW

File: lib/control/validation/fetch.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const HTTP_BAD_REQUEST = 400;
```
Comment (fmitchell-r7) [inline lib/control/validation/fetch.js]: I think we have a couple of constants for HTTP response codes scattered through the various validation modules. Do we want to consolidate those in a single place? I wouldn't try to do that in this pull request. Just an observation for a future refactor.

Comment (rvanburen-r7) [inline lib/control/validation/fetch.js]: you're right. I should probably extract all the BAD_REQUEST responses into a separate module like with the method not allowed module.

Comment (rvanburen-r7) [inline lib/control/validation/fetch.js]: Moving bad request to own module is now PR #27

**Comment by fmitchell-r7** [comment]
👍  Thanks for stubbing this out. 🚢  it.

**Pull request merged**